server/utils/constants.js

```diff
@@ -1 +1,33 @@
+import jwt from 'jsonwebtoken';
+
 export const TIMESTAMP = 'YYYY-MM-DD HH:mm:ss';
+
+export class Token {
+  static secret = 'express-graphql-template-v0';
```

Comment:
get the secret from the environment variable, inject the secret from github secrets.

Reply:
please create another `Tokens` file for this